react-interactive/src/GenericSlice.tsx

```diff
@@ -317,23 +317,24 @@ export default class GenericSlice<T extends U> {
             return await handle
         });
 
-        const dBPage = createAsyncThunk(`${name}/Page${name}`, async (args: { filter:  Search.IFilter<T>[], sortField?: keyof T, ascending?: boolean, page?: number}, { getState, signal }) => {
+        const dBPage = createAsyncThunk(`${name}/Page${name}`, async (args: { filter: Search.IFilter<T>[], sortField?: keyof T, ascending?: boolean, page?: number, parentID?: number | string }, { getState, signal }) => {
 
             const state = (getState() as any)[name] as IPagedState<T>;
             if (this.actionDependency !== null)
-                this.actionDependency(state,`${name}/Page${name}`, args)
+                this.actionDependency(state, `${name}/Page${name}`, args)
 
             let sortfield = args.sortField;
             let asc = args.ascending;
             const page = args.page ?? 0;
+            const parentID = args.parentID ?? undefined;
 
             sortfield = sortfield === undefined ? state.PagedSortField : sortfield;
             asc = asc === undefined ? state.PagedAscending : asc;
 
             if (this.pageHandle != null && this.pageHandle.abort != null)
                 this.pageHandle.abort('Prev');
 
-            const handle = this.controller.PagedSearch(args.filter, sortfield, asc, page, state.ParentID ?? undefined);
+            const handle = this.controller.PagedSearch(args.filter, sortfield, asc, page, parentID);
```

Comment:
This change actually breaks existing functionality. Previously if a Component was using a slice and set the `ParentID` (e.g. via `fetch`, that `parentID` was respected by `dBPage`. By making this branch that breaks and any component that calls `Fetch(parentID....)` and then `dBPage()` will no longer  work correctly